src/implementations/vanilla/components/gallery/gallery.scss

```diff
@@ -15,6 +15,29 @@ $_image-height: 185px;
 $_description-background: map.get(color.$color, 'grey-100');
 $_description-color: map.get(color.$color, 'white-100');
 
+@mixin ecl-aspect-ratio($width, $height) {
```

Comment:
Here I needed to define a `mixin` which is already declared in the media-conainer component:

https://github.com/ec-europa/europa-component-library/blob/4092b3aadcf87e148b679aee92bede0a0ad794d0/src/implementations/vanilla/components/media-container/media-container.scss#L11

Where can I put this `mixin` to make it global, create a new package?

FI, there is also an existing `mixin` in the utilities, but which does not take the iframe into account:
https://github.com/ec-europa/europa-component-library/blob/4af96b17a5d9ab4a2a5e6e2dbbdd04d27889de04/src/implementations/vanilla/utilities/media/media.scss#L6

Reply:
Good finding that there is an existing mixin in utilities. This is the right global place to put items to be shared between components and website.

Looking briefly at the source code, I think the best approach would be to add the iframe specifics into the utility and then use this utility in the media container and this component.

Reply:
On a separate note, there is no need to have `ecl-` namespaces at variables and mixins in v3 branch anymore, as we use the `@use` feature to import files which are treated as modules and there are no naming clashes.

I.e. v2 uses libsass and there are no modules, whereas v3 uses dart-sass and uses modules.

Reply:
Super @kalinchernev many thanks for the clarifications !